### PR TITLE
Refactor manifests into base and bsp

### PR DESCRIPTION
**Pull request by DunderRoffe**
Separate pelux-intel.xml into pelux-base.xml and pelux-intel.xml. Where
"pelux-base.xml" speicifies things that will be included in every pelux build
and "pelux-intel.xml" includes "pelux-base.xml" and adds bsp specifics.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

**Comment by DunderRoffe** [comment]
I agree that we should move towards fixed revisions but that is out of scope for this pr.

As for syntactic correctness I already did test run this and it is currently being run by Jenkins as well :)

File: pelux-intel.xml [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-  <project remote="oe"     revision="krogoth" name="meta-openembedded"      path="sources/meta-openembedded"/>
-  <project remote="yocto"  revision="krogoth" name="poky"                   path="sources/poky"/>
-  <project remote="github" revision="master"  name="Pelagicore/meta-bistro" path="sources/meta-bistro" />
-  <project remote="yocto"  revision="krogoth" name="meta-virtualization"    path="sources/meta-virtualization"/>
-
   <!-- Target stuff -->
-  <project remote="yocto" revision="krogoth" name="meta-intel"                          path="sources/meta-intel"/>
+  <project remote="yocto"  revision="krogoth" name="meta-intel"                      path="sources/meta-intel"/>
   <project remote="github" revision="master"  name="Pelagicore/meta-pelux-bsp-intel" path="sources/meta-pelux-bsp-intel" />
   <project remote="github" revision="master"  name="Pelagicore/meta-pelux"           path="sources/meta-pelux" />
```
Comment (socec) [inline pelux-intel.xml]: Shouldn't meta-pelux be in pelux-base.xml?

Comment (DunderRoffe) [inline pelux-intel.xml]: Yes you are right. My bad

Comment (DunderRoffe) [inline pelux-intel.xml]: Fixed

**Pull request merged**